tests/tests/blocks.rs

```diff
@@ -423,4 +424,95 @@ mod full_block {
         assert_eq!(block.header.height.0, 1);
         assert_eq!(block.transactions.len(), 2 /* mint + our tx */);
     }
+
+    use fuel_core_types::{
+        fuel_asm::{
+            op,
+            RegId,
+        },
+        fuel_vm::SecretKey,
+    };
+
+    use rand::{
+        rngs::StdRng,
+        Rng,
+        SeedableRng,
+    };
+
+    #[tokio::test]
+    async fn too_many_transactions_are_split_in_blocks() {
+        let max_gas_limit = 50_000_000;
+        let mut rng = StdRng::seed_from_u64(2322);
```

Comment:
Just curious, why 2322? This is what I got from googling it https://www.sosyncd.com/the-meaning-of-the-2322-angel-number/ :blush:

Reply:
Not sure, I just copy-pasted from other tests :)

Reply:
but I like the `inner wisdom and strength` argument from the website :D

Reply:
Yeah, perhaps that's the origin of the number :sweat_smile: